specification/keyvault/Security.KeyVault.EKM/models.tsp

```diff
@@ -0,0 +1,85 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+namespace KeyVault;
+
+/**
+ * A EkmConnection model object.
+ */
+model EkmConnection {
+  /**
+   * EKM proxy FQDM (Fully Qualified Domain Name). Only allowed characters are a-z, A-Z, 0-9, hyphen (-), dot (.), and colon (:).
+   */
+  host: string;
+
+  /**
+   * Optional path prefix for the EKM proxy (if any).
+   */
+  @encodedName("application/json", "path_prefix")
+  pathPrefix?: string;
+
+  /**
+   * The root CA certificate chain that issued the proxy server's certificate. An array of certificates in the certificate chain, each in DER format and base64 encoded.
```

Comment:
1. Base64- or base64url-encoded? Most places in KV uses base64url, which is why I ask.
2. You needed to add `@encoded("base64")` (or `@endcoded("base64url")`) to this/these parameters then to make sure the code that is generated is correct.